examples/tracing/cache_miss.py

```diff
@@ -0,0 +1,123 @@
+#!/usr/bin/python
+#
+# cache_miss.py Example of using BPF perf event.
```

Comment:
Absolutely! This can be a great tool. I'm actually thinking this can be a generalized tool for multiple kinds of events?